### DISCUSSION TITLE
Hotkey for "Connect" on query tab?

### DISCUSSION THREAD

**Discussion by arvinyorro** [microsoft/azuredatastudio#15142] [Q&A]
Any chance there is a hotkey for the connect button on the query tab?

![image](https://user-images.githubusercontent.com/1425668/114796605-ff291c80-9dc3-11eb-8f83-0de768f0343b.png)

**Comment by josefranciscorm** [comment]
Yes, please! This would be so helpful. Or add a command similar to the MSSQL extension for VS Code where you can select which connection to use.
![image](https://user-images.githubusercontent.com/16693363/146191321-35a504b0-a5db-49ad-884c-72fcbb0d1c15.png)

**Reply by austinbarberrepo** [reply]
I know this is an old thread, but you could create your own keybinding like in VSCode. When I press alt+c it brings up the connection screen as you would when you press connect. I was trying to see if I could add a connectionString in there to connect to a specific database but haven't come across that yet. But this seems to be similar behavior like VSCode.

// Place your key bindings in this file to override the defaults
[
    {
        "key": "alt+c",
        "command": "azdata.connect",
        "when": "editorTextFocus && editorLangId == 'sql'"
      },
]

**Comment by erinstellato-ms** [comment]
Oh hey @nocomputeruser, thanks for sharing this suggestion!  It's likely the best option here, as different users may have a preferred hotkey combination.

That said, @arvinyorro, if you believe there should be a default hotkey in ADS for this, could you please file a request in the Issues tab (https://github.com/microsoft/azuredatastudio/issues?q=is%3Aissue+is%3Aopen+)?  I did a quick search and don't see anything similar that already exists.  Thanks!
